test/e2e/specs/editor/various/font-size-picker.spec.js

```diff
@@ -8,6 +8,16 @@ test.describe( 'Font Size Picker', () => {
 		await admin.createNewPost();
 	} );
 
+	test.afterEach( async ( { page } ) => {
+		const closeButton = await page.locator(
```

Comment:
locator doesn't return a promise. It always returns a locator instance which evaluates itself lazily.

Reply:
The locator was fixed :+1: